scalingo/main.go

```diff
@@ -127,14 +129,24 @@ func main() {
 			}
 			return
 		}
-
-		defer update.Check()
 	} else {
 		// If we are completing stuff, disable logging
 		config.C.DisableInteractive = true
 	}
 
-	if err := app.RunContext(ctx, os.Args); err != nil {
+	err := app.RunContext(ctx, os.Args)
+	if err != nil {
 		fmt.Println("Fail to run command:", err)
 	}
+
+	// We want to display to the user if a new version is available
+	// Whatever the success of the execution of their command is.
+	updateCheckErr := update.Check()
+	if err != nil {
```

Comment:
**issue:** isn't it updateCheckErr you should check?

Reply:
You're 100% right